### PR TITLE
Add PHP Unit test framework CVE-2017-9841

### PR DESCRIPTION
**Pull request by mazzma12**

### CODE REVIEW

File: scenarios/crowdsecurity/CVE-2017-9841.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    Lower(evt.Meta.http_path) endsWith 'util/php/eval-stdin.php'
+blackhole: 1m
+groupby: "evt.Meta.source_ip"
+labels:
+  type: exploit
+  remediation: true
+  classification:
+    - attack.T1595
+    - attack.T1190
+    - cve.CVE-2023-22518
```
Comment (blotus) [inline scenarios/crowdsecurity/CVE-2017-9841.yaml:16]: Please fix the CVE number

File: scenarios/crowdsecurity/CVE-2017-9841.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+## CVE-2017-9841
+
+Detects exploit of CVE-2017-9841 vulnerability targeting PHP unit test framework.
+
+:warning: Smart attackers can exploit this vulnerability without leaving traces in logs
```
Comment (blotus) [inline scenarios/crowdsecurity/CVE-2017-9841.md:5]: Remove this line.

**Review by blotus — CHANGES_REQUESTED** [review]

File: scenarios/crowdsecurity/CVE-2017-9841.md (hunk 2 above)
Comment (mazzma12) [reply scenarios/crowdsecurity/CVE-2017-9841.md:5]: sure, was an old copy paste

File: scenarios/crowdsecurity/CVE-2017-9841.yaml (hunk 1 above)
Comment (mazzma12) [reply scenarios/crowdsecurity/CVE-2017-9841.yaml:16]: thx

**Review by blotus — APPROVED** [review]

**Pull request merged**